Packages/DV3D/ButtonBarWidget.py

```diff
@@ -442,7 +465,21 @@ def init( self, build_args, **args ):   # ( "Step", ("Run","Stop") ), self.proce
                 self.addButton( bspec[0], toggle=bspec[1], **args ) 
             else: 
                 self.addButton( bspec, **args )
-            
+
+    def changeButtonActivations(self, activation_list ):
+#        print " ** Change Button Activations: ", str( activation_list )
```

Comment:
I'll stop spamming notifications on these, but let's make a pass at cleaning up the rest of the commented-out code before merging.